src/qt/glcanvas.cpp

```diff
@@ -16,43 +16,27 @@
 #include <QtWidgets/QGestureRecognizer>
 #include <QtWidgets/QGestureEvent>
 
-#if defined(__VISUALC__)
-    #pragma message("OpenGL support is not implemented in wxQt")
-#else
-    #warning "OpenGL support is not implemented in wxQt"
-#endif
 wxGCC_WARNING_SUPPRESS(unused-parameter)
 
 class wxQtGLWidget : public wxQtEventSignalHandler< QGLWidget, wxGLCanvas >
 {
 public:
     wxQtGLWidget(wxWindow *parent, wxGLCanvas *handler, QGLFormat format)
-        : wxQtEventSignalHandler<QGLWidget,wxGLCanvas>(parent, handler)
-        {
-            setFormat(format);
-            setAutoBufferSwap( false );
-        }
+        : wxQtEventSignalHandler<QGLWidget, wxGLCanvas>(parent, handler)
+    {
+        setFormat(format);
+        setAutoBufferSwap(false);
+        setFocusPolicy(Qt::StrongFocus);
```

Comment:
BTW, I find it surprising that we don't call `setFocusPolicy()` at `wxWindow` level, it ought to take into account the return values of `AcceptsFocus()` and `AcceptsFocusFromKeyboard()` to determine the correct value to call it with.

This doesn't prevent this PR from being merged, of course, it just seems strange to do this only for the OpenGL canvas but not the other windows.